packetery/src/Packetery/Carrier/Carrier.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * Class Carrier
+ *
+ * @package Packetery\Carrier
+ */
+
+namespace Packetery\Carrier;
+
+/**
+ * Class Carrier
+ *
+ * @package Packetery\Carrier
+ */
+class Carrier {
```

Comment:
přejmenovat z Packetery\Carrier\Carrier na Packetery\Carrier\Entity. Ať je to jednotné s Packetery\Order\Entity